server/jetstream_cluster.go

```diff
@@ -6468,6 +6468,9 @@ func (mset *stream) checkAllowMsgCompress(peers []string) {
 	mset.mu.Unlock()
 }
 
+// To warn when we are getting tooo far behind from what has been proposed vs what has been committed.
```

Comment:
typo `too`

Reply:
Will fix and then merge once green. Thanks.